manipulator/arm_joint:
  controller:
    package: manipulator  
    module: manipulator_joint_controller
    type: ManipulatorJointController
  joint_name: arm
  joint_speed: 1.0
  joint_torque_limit: 1.0
  motor:
    id: 10
    cw_limit: 0
    ccw_limit: 6.28165

manipulator/wrist_joint:
  controller:
    package: manipulator
    module: manipulator_joint_controller
    type: ManipulatorJointController
  joint_name: wrist
  joint_speed: 1.0
  joint_torque_limit: 1.0
  motor:
    id: 11
    cw_limit: 0.2
    ccw_limit: 3.44

manipulator/hand_joint:
  controller:
    package: manipulator
    module: manipulator_joint_controller
    type: ManipulatorJointController
  joint_name: hand
  joint_speed: 1.0
  joint_torque_limit: 1.0
  motor:
    id: 12
    cw_limit: .5
    ccw_limit: 2.7
